src/tcp/flow.mli

```diff
@@ -20,4 +20,10 @@ module Make (IP:Tcpip.Ip.S)
             (R:Mirage_random.S) : sig
   include Tcpip.Tcp.S with type ipaddr = IP.ipaddr
   val connect : IP.t -> t Lwt.t
+
+  (**/**)
+  (* the number of open connections *)
+  val internal_n_channels : t -> int
```

Comment:
A bit of bike-shedding but why not just `num_channels` ? or even `num_open_channels`